types/relay-runtime/lib/subscription/requestSubscription.d.ts

```diff
@@ -3,18 +3,24 @@ import { GraphQLTaggedNode } from '../query/RelayModernGraphQLTag';
 import { Variables, Disposable } from '../util/RelayRuntimeTypes';
 import { SelectorStoreUpdater, Environment } from '../store/RelayStoreTypes';
 
-export interface GraphQLSubscriptionConfig<TSubscriptionPayload> {
+export interface SubscriptionParameters {
+    readonly response: {};
+    readonly variables: {};
+    readonly rawResponse?: {};
+}
+
+export interface GraphQLSubscriptionConfig<TSubscription extends SubscriptionParameters> {
```

Comment:
Can we use `OperationType` here instead of  `SubscriptionParameters`?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ee67bcaea06bc3d47dcbcdf0e2c1ed6d56319bfd/types/relay-runtime/lib/util/RelayRuntimeTypes.d.ts#L17-L21

```suggestion
export interface GraphQLSubscriptionConfig<TSubscription extends OperationType> {
```